tests/tck-build-logic/src/main/groovy/org/graalvm/internal/tck/harness/tasks/CheckstyleInvocationTask.groovy

```diff
@@ -22,7 +23,7 @@ abstract class CheckstyleInvocationTask extends AbstractSubprojectTask {
 
     @Inject
     CheckstyleInvocationTask(String coordinates, List<String> cmd) {
```

Comment:
If we removed String from constructors everywhere, should we also remove it here?